functions/api/unit.js

```diff
@@ -0,0 +1,135 @@
+const constants = require('../config/constants')
+const validate = require('../validation/validation')
```

Comment:
can we rename the file to validateSchema?

Reply:
Will change it to validator or it will be validateSchema.validateParams